src/engine/engine_wb.c

```diff
@@ -97,6 +97,9 @@ static void _ocf_write_wb_complete(struct ocf_request *req, int error)
 		req->error |= error;
 	}
 
+	if (req->error == 0)
```

Comment:
this must be moved to after line 104 - otherwise you will flush status bits for all sectors before all data I/O is finished

Reply:
done